pkg/controller/job/job_controller.go

```diff
@@ -1874,6 +1914,13 @@ func backoffLimitMetricsLabel(job *batch.Job) string {
 	return "global"
 }
 
+func jobFinishedNumReasonLabel(job *batch.Job) string {
```

Comment:
If we defer metrics this is not needed for now

Reply:
Done.